react/features/contact-list/components/ContactListPanel.web.js

```diff
@@ -47,46 +38,18 @@ class ContactListPanel extends Component {
     };
 
     /**
-     * Initializes a new {@code ContactListPanel} instance.
-     *
-     * @param {Object} props - The read-only properties with which the new
-     * instance is to be initialized.
-     */
-    constructor(props) {
-        super(props);
-
-        // Bind event handler so it is only bound once for every instance.
-        this._onOpenInviteDialog = this._onOpenInviteDialog.bind(this);
-    }
-
-    /**
      * Implements React's {@link Component#render()}.
      *
      * @inheritdoc
      */
     render() {
-        const { _locked, _participants, t } = this.props;
+        const { _participants, t } = this.props;
 
         return (
             <div className = 'contact-list-panel'>
                 <div className = 'title'>
                     { t('contactlist', { pcount: _participants.length }) }
                 </div>
-                <div className = 'sideToolbarBlock first'>
-                    <Button
-                        appearance = 'primary'
-                        className = 'contact-list-panel-invite-button'
-                        id = 'addParticipantsBtn'
-                        onClick = { this._onOpenInviteDialog }
-                        type = 'button'>
-                        { t('addParticipants') }
```

Comment:
"addParticipants" doesn't look to be used anymore and can be removed from translations?